UI/MainUI/Kazakh.lproj/Localizable.strings

```diff
@@ -76,7 +76,7 @@
 "Welsh" = "Cymraeg";
 
 "About" = "Жүйе туралы";
-"AboutBox" = "Developed by Inverse and Alinto, SOGo is a fully-featured groupware server with a focus on scalability and simplicity.<br/><br/>\nSOGo provides a rich AJAX-based Web interface and supports multiple native clients through the use of standard protocols such as CalDAV and CardDAV.<br/><br/>\nSOGo is distributed under the <a href=\"http://gnu.org/licenses/gpl.html\">GNU GPL</a> version 2 or later and parts are distributed under the GNU LGPL version 2. This is free software: you are free to change and redistribute it. There is NO WARRANTY, to the extent permitted by law.<br/><br/>\nSee <a href=\"http://www.sogo.nu/en/support/community.html\">this page</a> for various support options.";
```

Comment:
Hi, this shall be translated in [Transifex](https://app.transifex.com/alinto/sogo)

Reply:
Hi, done!